### PR TITLE
Reset the robot upon power on.

### PR DESCRIPTION
**Pull request by deestroy**
Resets the robot and turns on beep to alert robot that the arm is active. Empty file for camera multithreading is put as a place holder while the camera is being fixed.

### CODE REVIEW

File: robot-endpoint/camera.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    # Display the resulting frame
+    cv.imshow('frame', gray)
+    if cv.waitKey(1) == ord('q'):
+        break
+# When everything done, release the capture
+cap.release()
+cv.destroyAllWindows()
+
+
+# import cv2
```
Comment (sarahchoww) [inline robot-endpoint/camera.py:30]: is this commented out for future use? or was this an old implementation

**Review by sarahchoww — APPROVED** [review]
looks good!

File: robot-endpoint/camera.py (hunk 1 above)
Comment (deestroy) [reply robot-endpoint/camera.py:30]: This was working in an old version but suddenly stopped working

**Review by sarahchoww — APPROVED** [review]
lgtm

**Pull request merged**